vm/binding/lib.rs

```diff
@@ -67,7 +84,14 @@ impl VM {
 
     /// Gets the next page of bytecode to be executed.
     pub fn get_next_page(&self) -> ContentId {
-        ContentId(unsafe { moshell_vm_next_page(self.0) })
+        ContentId(unsafe { moshell_vm_next_page(self.ffi) })
+    }
+
+    pub fn get_exported_var(&self, name: &str) -> VmValueFFI {
+        unsafe {
+            let c_name = CString::new(name).unwrap();
+            moshell_vm_get_exported(self.ffi, c_name.into_raw())
```

Comment:
```suggestion
            moshell_vm_get_exported(self.ffi, c_name.as_ptr())
```
The string is still owned by this function (or even better, don't take a C-string in the FFI by using a pointer and a size).